src/store/user/slice.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const userSlice = createSlice({
+  name: "user",
+  initialState: {
+    status: "no data",
```

Comment:
maybe it's better to use the statuses from a defined list via constants without writing them every time directly